components/ILIAS/UI/src/Implementation/Component/MessageBox/EntityListingRendererFactory.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+namespace ILIAS\UI\Implementation\Component\MessageBox;
+
+use ILIAS\UI\Implementation\Render;
+
+class EntityListingRendererFactory extends Render\DefaultRendererFactory
```

Comment:
As far as I can see, this renderer is not yet used, correct?

Can you confirm that we need to implement it if we want to render the entity list differently?

Reply:
A new renderer might not be necessary, we will see
I will delete this for now in the next commit